src/EditorFeatures/Core/Implementation/IntelliSense/SignatureHelp/Controller.Session_SetModelSelectedItem.cs

```diff
@@ -19,8 +19,7 @@ private void SetModelExplicitlySelectedItem(Func<Model, SignatureHelpItem> selec
                 AssertIsForeground();
 
                 Computation.ChainTaskAndNotifyControllerWhenFinished(
-                    model => SetModelExplicitlySelectedItemInBackground(model, selector),
-                    updateController: false);
+                    model => SetModelExplicitlySelectedItemInBackground(model, selector));
```

Comment:
so this code is very dusty in my brain and i don't reacall all the nuances of it.  now that you're removing this value (and seemingly updating on every task in the chain) should we et rid of the "AndNotifyWhenFinished" part of the method name (since it seems like we always notify or each task in the chain)?

Reply:
The "AndNotifyWhenFinished" part of the name seems more correct now than it was before. Previously it would only _sometimes_ notify the controller when finished.